templates/home.html

```diff
@@ -75,6 +75,14 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
             <span class="ml small">അറിയിപ്പുകൾ</span>
         </span>
     </a>
+    <a href="https://www.facebook.com/IndiaHSI/" class="home-button card" role="button">
```

Comment:
Suggestion : As its an outgoing link from the webpage , should this be on a new tab or page?
Something like this.
`<a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/IndiaHSI/">`